e2e/pages/FilePickerPage.ts

```
@@ -71,6 +71,10 @@ export class FilePickerPage {
       .waitFor({ state: 'visible', timeout: 10_000 })
   }
 
+  async doubleClickItem(name: string): Promise<void> {
+    await this.getListItemByName(name).getByTestId('listitem-onclick').dblclick()
```

ESLint

[error] Replace `.getByTestId('listitem-onclick')` with `⏎······.getByTestId('listitem-onclick')⏎······`

(prettier/prettier)